plugin/pkg/admission/limitranger/admission_test.go

```diff
@@ -634,3 +642,64 @@ func TestLimitRangerCacheAndLRUExpiredMisses(t *testing.T) {
 		t.Errorf("Should have ignored calls to any subresource of pod %v", err)
 	}
 }
+
+func validPersistentVolumeClaim(name string, resources api.ResourceRequirements) api.PersistentVolumeClaim {
+	pvc := api.PersistentVolumeClaim{
+		ObjectMeta: api.ObjectMeta{Name: name, Namespace: "test"},
+		Spec: api.PersistentVolumeClaimSpec{
+			Resources: resources,
+		},
+	}
+	return pvc
+}
+
+func TestPersistentVolumeClaimLimitFunc(t *testing.T) {
+	type testCase struct {
+		pvc        api.PersistentVolumeClaim
+		limitRange api.LimitRange
+	}
+
+	successCases := []testCase{
+		{
+			pvc:        validPersistentVolumeClaim("pvc-is-min-storage-request", getResourceRequirements(getStorageResourceList("1Gi"), getStorageResourceList(""))),
+			limitRange: createLimitRange(api.LimitTypePersistentVolumeClaim, getStorageResourceList("1Gi"), api.ResourceList{}, api.ResourceList{}, api.ResourceList{}, api.ResourceList{}),
```

Comment:
@derekwaynecarr regarding your comment about using `api.LimitTypePersistentVolumeClaim` in  a test, doesn't this do it? The `LimitRange` is created w/ this type and then `PersistentVolumeClaimLimitFunc` looks specifically for this type.